dev/io.openliberty.io.smallrye.reactive.messaging-provider4/src/io/openliberty/microprofile/reactive/messaging30/internal/OLReactiveMessaging30Extension.java

```diff
@@ -97,20 +109,55 @@ void beforeBeanDiscovery(@Observes BeforeBeanDiscovery discovery, BeanManager be
 
     @Override
     protected void afterDeploymentValidation(@Observes AfterDeploymentValidation done, BeanManager beanManager) {
-        // Catch DeploymentException that is thrown if there are any validation errors with the Reactive Messaging Configuration
-        try {
-            MediatorManager mediatorManager = configureMediatorManager(beanManager);
+        final ContextBeginnerEnder contextBeginnerEnder = createContextBeginnerEnderIfCheckpointEnabled();
+        final MediatorManager mediatorManager = configureMediatorManager(beanManager); //TODO I only fonud one DeploymentException in the small rye source so there's no need for a second catch (DeploymentException) block. But double check this with the team. 
```

Comment:
We should only get the DeploymentExceptions from the startMediatorManager call. not during the configure section, so given that is covered in the startMediatorManager method and as long as the TCK passes then our requirements are satisfied.